cmd/kube-proxy/app/init_windows.go

```diff
@@ -19,16 +19,46 @@ limitations under the License.
 package app
 
 import (
-	"k8s.io/kubernetes/pkg/windows/service"
+	"fmt"
 
 	"github.com/spf13/pflag"
+	"golang.org/x/sys/windows"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/windows/service"
 )
 
 const (
 	serviceName = "kube-proxy"
 )
 
-func initForOS(windowsService bool) error {
+// getPriorityValue returns the value associated with a Windows process priorityClass
```

Comment:
Maybe (just suggesting) this can be moved to "k8s.io/kubernetes/pkg/windows/service" into a new 'utils' or whatever file there?

As kubelet also imports this, this can be shared with Kubelet :)

Reply:
Do you mean `getPriorityValue` function?